packages/components/src/DataList/components/DataListLayoutInternal/index.ts

```diff
@@ -0,0 +1,2 @@
+export { DataListItems } from "./DataListItems";
+export { DataListHeader } from "./DataListHeader";
```

Comment:
don't really know how I feel about these 2 files being exported in a totally different named folder. It is an internal thing tho, so maybe that's okay.

Reply:
As they're internal components from this component, I think it's okay. I extracted the props to an interface tho